### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by rhatdan**
Add missing kpod-ps information
Add link for kpod history video.

Signed-off-by: Daniel J Walsh <dwalsh@redhat.com>

### CODE REVIEW

**Comment by rhatdan** [comment]
@mheon @umohnani8 PTAL

File: README.md [hunk 1]
```diff
@@ -44,13 +44,14 @@ It is currently in active development in the Kubernetes community through the [d
...
 | [kpod-images(1)](/docs/kpod-images.1.md)             | List images in local storage                                              |[![...](/docs/play.png)](https://asciinema.org/a/133649)|
 | [kpod-info(1)](/docs/kpod-info.1.md)                 | Display system information                                                ||
 | [kpod-inspect(1)](/docs/kpod-inspect.1.md)           | Display the configuration of a container or image                         |[![...](/docs/play.png)](https://asciinema.org/a/133418)|
 | [kpod-load(1)](/docs/kpod-load.1.md)                 | Load an image from docker archive or oci                                  |[![...](/docs/play.png)](https://asciinema.org/a/kp8kOaexEhEa20P1KLZ3L5X4g)|
 | [kpod-logs(1)](/docs/kpod-logs.1.md)                 | Display the logs of a container                                           ||
 | [kpod-mount(1)](/docs/kpod-mount.1.md)               | Mount a working container's root filesystem                               ||
+| [kpod-ps(1)](/docs/kpod-ps.1.md)                     | Prints out information about containersregistry                           ||
```
Comment (TomSweeneyRedHat) [inline README.md]: "containersregistry" to "containers registry"

Comment (TomSweeneyRedHat) [inline README.md]: or do we want to be consistent with the other descriptions and go to "Prints out information about containers"

**Comment by TomSweeneyRedHat** [comment]
One typo, otherwise LGTM.  Urvashi did a nice demo yesterday on this, it would be good to capture similar in asciinema.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Trust me this is a big one to get out for @umohnani8, we need a video and blog.

**Comment by rhatdan** [comment]
@TomSweeneyRedHat Fixed the conainersregistry thing.

**Comment by TomSweeneyRedHat** [comment]
LGTM, 👍 on the blog/video thought.  🥇 to Urvashi's demo yesterday, if the blog/video is half as good as that it will still rock.

**Comment by umohnani8** [comment]
LGTM. Thanks @TomSweeneyRedHat and I will get the asciinema/blog done today @rhatdan.

**Comment by mrunalp** [comment]
This can be merged. The Fedora test failure is an unrelated race. I'll work on a fix for it.

**Comment by mrunalp** [comment]
LGTM

**Pull request merged**